Cinema Fullstack

Ananiev-Vitalii opened the pull request:
Не можу закріпити скріни. Мають формат JPG але видає помилку:We don’t support that file type.

Try again with GIF, JPEG, JPG, MOV, MP4, PNG, SVG, WEBM, CPUPROFILE, CSV, DMP, DOCX, FODG, FODP, FODS, FODT, GZ, JSON, JSONC, LOG, MD, ODF, ODG, ODP, ODS, ODT, PATCH, PDF, PPTX, TGZ, TXT, XLS, XLSX or ZIP.

Vitalii-A committed:
Cinema Fullstack.

Vitalii-A committed:
Cinema Fullstack

vsmutok reviewed (changes requested):
NOTE: Attach screenshots of all pages from the correctly connected frontend. Better to make them with opened developer tool, where will be shown requests to the API.

vsmutok commented:
> Не можу закріпити скріни. Мають формат JPG але видає помилку:We don’t support that file type.
> 
> Try again with GIF, JPEG, JPG, MOV, MP4, PNG, SVG, WEBM, CPUPROFILE, CSV, DMP, DOCX, FODG, FODP, FODS, FODT, GZ, JSON, JSONC, LOG, MD, ODF, ODG, ODP, ODS, ODT, PATCH, PDF, PPTX, TGZ, TXT, XLS, XLSX or ZIP.

конвертуй фото в потрібний формат)

Vitalii-A commented:
![5](https://github.com/user-attachments/assets/fb8ad512-d5fb-4af4-8119-17365cdaeaaa)
![6](https://github.com/user-attachments/assets/d36a551a-8f3f-4033-94f0-ecd7f40d4ee8)
![7](https://github.com/user-attachments/assets/b7cf0016-0555-48ca-b7a4-0326be9bddc1)
![8](https://github.com/user-attachments/assets/76c67f12-a573-40e9-96e9-102ef8c85b71)
![9](https://github.com/user-attachments/assets/31a28f49-841c-4a0c-9d2b-7809ef100711)
![10](https://github.com/user-attachments/assets/7f627d62-7b9d-4286-9841-7305eba2c599)
![14](https://github.com/user-attachments/assets/d3e328b7-e18b-47ea-a31f-e6aa26fcbc0a)
![13](https://github.com/user-attachments/assets/d4bd2d2b-d20b-47d0-b9f9-62ec45b97988)
![12](https://github.com/user-attachments/assets/1d738208-d143-46b5-8ee3-c8a6823ab018)
![1](https://github.com/user-attachments/assets/18cccaf1-3135-464e-8002-4981d4389264)
![11](https://github.com/user-attachments/assets/8ad88a59-12f4-45d0-a41d-f48ae3d86d25)
![2](https://github.com/user-attachments/assets/35c2e3d8-71cd-49be-a461-eeb63332bb01)
![3](https://github.com/user-attachments/assets/8bba81fd-dd98-411d-83c5-873f21c52544)
![4](https://github.com/user-attachments/assets/67376d4a-2736-4259-a0ee-ecbe55829f12)

vsmutok reviewed (changes requested):
Attach screenshots showing that the images are displayed correctly on website.

Ananiev-Vitalii commented:
> Attach screenshots showing that the images are displayed correctly on website.

![1](https://github.com/user-attachments/assets/345a0d9c-4ebf-4f8d-bf29-d5cc281d831f)
![2](https://github.com/user-attachments/assets/5a331758-2d17-474a-90f8-3fa8242ec7b4)
![3](https://github.com/user-attachments/assets/b8758676-c439-40e0-b4ba-59e1ec7f86ea)
![4](https://github.com/user-attachments/assets/12a6f6e0-8097-40c6-9477-310198b79c11)
![5](https://github.com/user-attachments/assets/3bbdc8b6-74dd-40bd-a5f6-0e8c7b69d2ba)
![6](https://github.com/user-attachments/assets/c73630da-a896-40d1-955d-2e46982a4610)
![7](https://github.com/user-attachments/assets/ce1d2650-fd97-447a-8a3a-d8d1dce11bf3)
![8](https://github.com/user-attachments/assets/0dc2dd8d-0725-4bfe-bfbd-c6543961a950)
![9](https://github.com/user-attachments/assets/8ee35a4b-fff7-42cb-8243-7ae2cb3f727e)
![10](https://github.com/user-attachments/assets/d6dc7841-9b5c-4496-8334-5c3db5964805)
![11](https://github.com/user-attachments/assets/1b953d04-f197-4614-9ce3-b7e78ac01f8a)
![12](https://github.com/user-attachments/assets/384f3a83-3e93-4953-85fe-5498a6a78ce0)

vsmutok commented on backend/cinema_service/settings.py:
```diff
@@ -81,40 +86,43 @@
-        "USER": os.environ["POSTGRES_USER"],
-        "PASSWORD": os.environ["POSTGRES_PASSWORD"],
+        "ENGINE": "django.db.backends.sqlite3",
+        "NAME": BASE_DIR / "db.sqlite3",
```
Why did you replace Postgres with SQLite? 
Please switch back to Postgres.

Vitalii-A committed:
- fix: adjust media/endpoint paths so images render correctly
- chore: remove SQLite from settings.py and switch to PostgreSQL (ENV-based config)

Vitalii-A committed:
chore: remove SQLite from settings.py and switch to PostgreSQL (ENV-based config)

Vitalii-A committed:
Merge remote-tracking branch 'origin/develop' into develop

# Conflicts:
#	backend/cinema_service/settings.py
#	frontend/src/views/MovieDetailsScreen.vue

vsmutok commented on backend/cinema_service/settings.py:
It looks like you forgot to add CORS settings to `settings.py`.
Without them, you’ll encounter errors.

Vitalii-A committed:
Added missing CORS settings in settings.py to fix related errors.
Also corrected the file path in MovieDetailsScreen.vue.
